### PR TITLE
Add code-deploy orb example to deploy integrations

### PR DESCRIPTION
**Pull request by teesloane**
follows up on [jira-12489](https://circleci.atlassian.net/secure/RapidBoard.jspa?rapidView=67&modal=detail&selectedIssue=CIRCLE-12489)

### CODE REVIEW

File: jekyll/_cci2/deployment-integrations.md [hunk 1]
```diff
@@ -192,9 +192,30 @@ workflows:
           container-image-name-updates: 'container=${MY_APP_PREFIX}-service,tag=${CIRCLE_SHA1}'
 ```
 
-For more detailed information about the AWS ECS & AWS ECR orbs, refer to the following Orb registry pages:
+
+##### AWS CodeDeploy
+
+This orb enables you to run deployments through AWS CodeDeploy
```
Comment (michelle-luna) [inline jekyll/_cci2/deployment-integrations.md]: add punctuation

Comment (michelle-luna) [reply jekyll/_cci2/deployment-integrations.md]: change "This" to The following or the `aws-code-deploy` orb

File: jekyll/_cci2/deployment-integrations.md [hunk 2]
```diff
@@ -192,9 +192,30 @@ workflows:
...
+
+```yaml
+version: 2.1
```
Comment (michelle-luna) [inline jekyll/_cci2/deployment-integrations.md]: can we add some comments to this example?

File: jekyll/_cci2/deployment-integrations.md (hunk 1 above)
Comment (teesloane) [reply jekyll/_cci2/deployment-integrations.md]: Done!

File: jekyll/_cci2/deployment-integrations.md (hunk 2 above)
Comment (teesloane) [reply jekyll/_cci2/deployment-integrations.md]: Yes! I added the orb parameter descriptions from the [corresponding documentation](https://circleci.com/orbs/registry/orb/circleci/aws-code-deploy)

**Pull request merged**